Source/WebCore/html/canvas/CanvasRenderingContext2DBase.cpp

```diff
@@ -1655,6 +1657,7 @@ ExceptionOr<void> CanvasRenderingContext2DBase::drawImage(Document& document, Ca
                 return { };
         }
         downcast<BitmapImage>(*image).updateFromSettings(document.settings());
+        shouldPostProcess = false;
```

Comment:
Maybe this should've been reposted for review.
If you look at the callers, this does not mean "the source is a .jpeg file".
The variants that end up here are  for example
`CanvasRenderingContext2DBase::drawImage(SVGImageElement& imageElement`.

Reply:
Can you explain why that is a concern? My understanding is that a `svg`'s `Image::isBitmapImage` won't return `true` because of the following code paths:

`SVGImageElement::cachedImage()` returns the result of `SVGImageLoader::image()`. `SVGImageLoader::image()` returns whatever object is in `ImageLoader::m_image` which is a `CachedImage`. `CachedImage::m_image` holds any `Image` derivative.

Next, `CachedImage::imageForRenderer()` returns either:

1. `CachedImage::m_image`,
1. the result of `SVGImageCache::imageForRenderer`, or
1. `Image::nullImage()`

Given that `SVGImage::isBitmapImage` returns `false`, all of these options seem safe within that conditional block. Is there a path that may return `true`?